components/ScheduleFilters/ScheduleFilters.tsx

```diff
@@ -180,137 +141,195 @@ const ScheduleFilters = ({ filters, handleFilters }: ISelectScheduleProps) => {
     });
   };
 
-  return (
-    <div className={styles.filters}>
-      {/* LEI */}
-      <Collapse
-        className={styles.checkbox}
-        bordered={false}
-        expandIcon={({ isActive }) => (
-          <CaretRightOutlined rotate={isActive ? 90 : 0} />
+  const checkedIndicator = (filter) => {
+    return (
+      <div className={styles.selected_schedules}>
+        {filter.shifts?.length ? (
+          filter.shifts.map((shiftOption) =>
+            isChecked({
+              id: filter.id,
+              shift: shiftOption,
+            }) ? (
+              <div className="ml-1 rounded-full bg-blue-200 p-1" />
+            ) : (
+              <></>
+            )
+          )
+        ) : isChecked({
+            id: filter.id,
+            shift: undefined,
+          }) ? (
+          <div className="ml-1 rounded-full bg-blue-200 p-1" />
+        ) : (
+          <></>
         )}
+      </div>
+    );
+  };
```

Comment:
Since this is a component, please ensure to use name convention with the first capitalized letter. Also, you can increase code readability by preventing to use this If-Else Hell with ternary operators and using Return Early Pattern.

```ts
  const CheckedIndicator = ({ filter }) => {
    if (isChecked({ id: filter.id, shift: undefined })) {
      return (
        <div className={styles.selected_schedules}>
          <div className="ml-1 rounded-full bg-blue-200 p-1" />
        </div>
      );
    }

    const isSomeChecked = filter.shifts?.some((shiftOption) =>
      isChecked({
        id: filter.id,
        shift: shiftOption,
      })
    );

    return (
      <div className={styles.selected_schedules}>
        {isSomeChecked && <div className="ml-1 rounded-full bg-blue-200 p-1" />}
      </div>
    );
  };
```